build.sh

```diff
@@ -42,10 +42,15 @@ PATH=$DEPOT_TOOLS_DIR:$DEPOT_TOOLS_DIR/python276_bin:$PATH
 mkdir -p $OUTDIR
 OUTDIR=$(readlink -f $OUTDIR)
 
+# This is a hack to force use of Microsoft Visual C++ 2013.
+# More sophisticated code would look first for Visual C++ 2015, then 2013.
+export GYP_MSVS_VERSION='2013'
+echo GYP_MSVS_VERSION $GYP_MSVS_VERSION
+
 set-platform
 clean $OUTDIR
-check::deps $PLATFORM $DEPOT_TOOLS_DIR
 check::depot-tools $PLATFORM $DEPOT_TOOLS_URL $DEPOT_TOOLS_DIR
+check::deps $PLATFORM $DEPOT_TOOLS_DIR
```

Comment:
The implication of putting ::deps after ::depot-tools is that ::depot-tools uses git would not be available yet. In a CI environment like travis/gitlab-ci this will fail the build.

Fortunately, looking things over again, we don't need jq anymore (it was a vestige from before when this repo would parse JSON to upload packages using the Github API)
https://github.com/vsimon/webrtcbuilds/pull/13

After this pull request is merged, check::depot_tools should now see that depot_tools is empty.